test-plugins/supervising_provider_plugin/CANVAS_MANIFEST.json

```diff
@@ -0,0 +1,29 @@
+{
+    "sdk_version": "0.1.4",
+    "plugin_version": "0.0.1a",
+    "name": "supervising_provider_plugin",
+    "description": "Edit the description in CANVAS_MANIFEST.json",
```

Comment:
Since we are committing this lets update the description to 

`Add annotations to the supervising provider dropdown in Prescribe, Refill, and Adjust Prescription Commands`